lib/resources/users.js

```diff
@@ -0,0 +1,26 @@
+const { endpoint } = require('../util/http');
+
+module.exports = (service, { User, Actor, Form }) => {
+
+  service.post('/users', endpoint(({ body }) =>
+    (new User(body)).create()
+  ));
+
+  service.post('/permstest', endpoint(() =>
+    /*Actor.getById(1).then((actor) =>
+      actor.grant([ 'read', 'create', 'update', 'delete', 'grant' ], 'user')
+    )*/
+    Actor.getById(40).then((actor) =>
+      actor.grant('whatever', 'user')
+    )
+  ));
+
+  service.get('/permstest/:verb/:formId', endpoint(({ params }) =>
+    Promise.all([
+      Actor.getById(1),
+      Form.getById(params.formId)
+    ]).then(([ actor, form ]) => actor.can(params.verb, form))
```

Comment:
And this is what most people would need to be able to use, right?

Reply:
Correct. `.can()`, `.grant()`, `.grantIf()` are probably the biggest API points. Very simple, and pretty much the same as the `node_acl` API as far as I saw.